src/components/ChoiceGrid/Styles.module.scss

```diff
@@ -234,12 +226,12 @@
   margin: 0 0 20px 0;
   text-align: center;
   @include font-size($text-large);
-  @include mq('tablet') {
+  @include mq("tablet") {
```

Comment:
Apologies for these changes caused by my linter. V annoying!

Reply:
My linter does the same thing! We should set up a linting config so we don't have to think about it. I'll try and have a look at this later.

Reply:
[Implementing Prettier](https://www.barrymcgee.co.uk/notes/format-your-code-prettier) would solve this issue once and for all. If you guys are interested, I can submit a PR?

Reply:
@barrymcgee hey man, yeah all contributions welcome!